srcpkgs/cnping/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cnping'
+pkgname=cnping
+version=1.0b2
+revision=1
+hostmakedepends="gcc make libcap-progs"
```

Comment:
`gcc` and `make` are part of the chroot, and `gcc` would be wrong if you want to cross compile.

Reply:
So just libcap-progs here?

Reply:
I think not even `libcap-progs`, because you would have to move `setcap` to the `INSTALL` script which runs when the package is installed on a users system.
Like `srcpkgs/iputils/INSTALL` and `srcpkgs/extrace/INSTALL`.

Reply:
then I need libcap-progs as a runtime dependency instead I assume?

Reply:
Or is there an option to depend on a package only during the installation?

Reply:
No there are no install only dependencies, `iputils` depends on `libcap-progs`, I think this is the way to go.

Reply:
Okay, will do that then. Thanks!